ArchiSteamFarm/Steam/Interaction/Commands.cs

```diff
@@ -3257,6 +3427,22 @@ internal void OnNewLicenseList() {
 		return responses.Count > 0 ? string.Join(Environment.NewLine, responses) : null;
 	}
 
+	private static HashSet<EAssetRarity>? ParseAssetRarities(string assetRaritiesText) {
+		string[] assetRaritiesArgs = assetRaritiesText.Split(SharedInfo.ListElementSeparators, StringSplitOptions.RemoveEmptyEntries);
+
+		HashSet<EAssetRarity> assetRarities = [];
+
+		foreach (string assetRarityArg in assetRaritiesArgs) {
+			if (!Enum.TryParse(assetRarityArg, true, out EAssetRarity assetRarity) || !Enum.IsDefined(assetRarity)) {
```

Comment:
I don't think `Enum.TryParse` can both return `true` and have an `out`-value that does not match `Enum.IsDefined(...)`.

Reply:
@Abrynos Check what happens when you try to parse e.g. `160` out-of-bound value, I think tryparse will succeed but you'll get value that isn't defined.

Reply:
You are right. I forgot you can parse integer-strings as enum and it will succeed.